### PR TITLE
Issue 15 (Part 1) - Last Check and Refactor

### PR DESCRIPTION
**Pull request by Rapter1990**
Description

- Revise OpenApiConfig file for Spring Boot Open Api
- Revise versions in pom.xml

### CODE REVIEW

File: src/test/java/com/project/parkinglot/config/OpenApiConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        Info info = OpenApiConfig.class.getAnnotation(OpenAPIDefinition.class).info();
+        Contact contact = info.contact();
+
+        // Then
+        assertEquals("Sercan Noyan Germiyanoğlu | Mehmet Şeymus Yüzen | Harun Yusuf Ekşioğlu | Muhammet Oğuzhan Aydoğan", contact.name());
+        assertEquals("https://github.com/Rapter1990/parkinglot/", contact.url());
+
+    }
+
+}
```
Comment (mehmetseymusyuzen) [inline src/test/java/com/project/parkinglot/config/OpenApiConfigTest.java:57]: abi bir satır bosluk bırakır misin?

**Review by mehmetseymusyuzen — CHANGES_REQUESTED** [review]

**Review by artfulCoder98 — APPROVED** [review]
LGTM

File: src/test/java/com/project/parkinglot/config/OpenApiConfigTest.java (hunk 1 above)
Comment (Rapter1990) [reply src/test/java/com/project/parkinglot/config/OpenApiConfigTest.java]: I revised it. Can you check it?

**Review by moaydogdu — APPROVED** [review]

**Review by anilhazar — APPROVED** [review]
LGTM

**Pull request merged**